app.py

```diff
@@ -1,43 +1,95 @@
-from flask import Flask, redirect, url_for, request, render_template
-from urllib.request import urlopen
+from flask import Flask, redirect, url_for, request, render_template, Response
+from flask_mail import Mail, Message
+import os
 import json
+import requests
+from multiprocessing import Pool
 
 # global variables to store repo and repo links
-repos=[]
-repolink=[]
+repos = []
+repolink = []
+image = ""
 
 # module to generate repo and repo links
+
+
 def repogen(user_name):
     global repos
     global repolink
+    global image
+    repos = []
+    repolink = []
     # getting contents of the api for a user
-    httpob=urlopen("https://api.github.com/users/"+user_name+"/repos")
-    # decoding the http object recieved
-    decob=httpob.read().decode("utf-8")
-    # converting to json
-    jsonob=json.loads(decob)
-    # appending only personal repos to the list
+    httpob = requests.get("https://api.github.com/users/" +
+                          user_name + "/repos?per_page=100")
+    owner = requests.get("https://api.github.com/users/" + user_name)
+    owner_data = json.loads(owner.content.decode("utf-8"))
+    decob = httpob.content.decode("utf-8")
+    jsonob = json.loads(decob)
+    image = jsonob[0]['owner']['avatar_url']
     for j in jsonob:
-        if str(j["fork"])=="False":
-            repos.append(j["full_name"])
-            repolink.append("https://www.github.com/"+j["full_name"])
+        if j["fork"] is not True:  # appending only personal repos to the list
+            repos.append(j["name"])
+            repolink.append(j["html_url"])
+
 
 app = Flask(__name__)
 
 
 @app.route('/<name>')
 def main(name):
-    global repos
-    global repolink
     # generating repo and repolink list before rendering the html file
-    # list to be passed while rendering & some more credentials can be added 
+    # list to be passed while rendering & some more credentials can be added
     repogen(name)
-    return render_template('test.html', name=name)
+    repo_dict = dict(zip(repos, repolink))
+    owner = requests.get("https://api.github.com/users/" + name)
+    owner_data = json.loads(owner.content.decode("utf-8"))
+    x = render_template('temp.html', name=name, image=image,
+                        repo_dict=repo_dict, owner_data=owner_data)
+    with open("templates/t2.html", "w") as f:
+        f.write(x)
+    headers = {"Access-Control-Allow-Origin": "*", 'Content-Type': 'text/html'}
+    return Response(x, 200, headers=headers)
+
+
+def send_mail(form_msg, form_name, form_email):
+    print("inside async")
+    app.config['MAIL_SERVER'] = 'smtp.gmail.com'
+    app.config['MAIL_PORT'] = 587  # 465 for SSL
+    app.config['MAIL_USE_TLS'] = True
+    app.config['MAIL_USE_SSL'] = False  # True
+    app.config['MAIL_USERNAME'] = 'dibyadasiscool@gmail.com'
+    app.config['MAIL_PASSWORD'] = 'samsunghp'
```

Comment:
Haha. This is a test email ID  I am using for development purposes. Once done, I will change that.